src/Numerics/Mesh/BrickMesh.jl

```diff
@@ -1086,5 +1086,44 @@ function connectmesh(
     )        # neighbor send ranges into `sendelems`
 end
 
+"""
+    (bndytoelem, bndytoface) = enumerateboundaryfaces!(elemtoelem, elemtobndy, periodicity, boundary)
+
+Update the `elemtoelem` array based on the boundary faces specified in
+`elemtobndy`. Builds the `bndytoelem` and `bndytoface` tuples.
+"""
+function enumerateboundaryfaces!(elemtoelem, elemtobndy, periodicity, boundary)
+    nb = 0
+    for i in 1:length(periodicity)
+        if !periodicity[i]
+            nb = max(nb, boundary[i]...)
+        end
+    end
+    # Limitation of the boundary condition unrolling in the DG kernels
+    # (should never be violated unless more general unstructured meshes are used
+    # since cube meshes only have 6 faces in 3D, and only 1 bcs is currently allowed
+    # per face)
+
+    @assert nb <= 6
+
+    bndytoelem = ntuple(b -> Vector{Int64}(), nb)
+    bndytoface = ntuple(b -> Vector{Int64}(), nb)
```

Comment:
```suggestion
    bndytoelem = ntuple(b -> Vector{Int}(), nb)
    bndytoface = ntuple(b -> Vector{Int}(), nb)
```

Reply:
True, but we hardcode everything else as `Int64`, so it should be consistent. I'll leave that for another PR.